addons/hr_timesheet/security/hr_timesheet_security.xml

```diff
@@ -52,8 +52,9 @@
             <field name="domain_force">[
                 ('user_id', '=', user.id),
                 ('project_id', '!=', False),
-                '|',
+                '|', '|',
                     ('project_id.privacy_visibility', '!=', 'followers'),
+                    ('partner_id', '=', user.partner_id.id),
```

Comment:
Actually, I am not sure to understand why we need to alter that rule. I think we just grant access to portal user? 🤔 

It seems strange to me an internal user will be set as customer in project/task. 🤷🏼‍♂️